unblob/extractor.py

```diff
@@ -85,12 +91,24 @@ def fix_symlink(path: Path, outdir: Path, task_result: TaskResult) -> Path:
 
 
 def fix_extracted_directory(outdir: Path, task_result: TaskResult):
+    def _fix_extracted_directory(directory: Path, task_result: TaskResult):
+        if not directory.exists():
+            return
+        for path in (directory / p for p in os.listdir(directory)):
```

Comment:
For the future: `os.scandir` (instead of `os.listdir`) would be a better match for the task, but it would require reworking `fix_permission` as well.

Reply:
Did not know about `scandir`, but yeah it makes sense to move to it in the future.